books/biology/bake

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '9.1.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :biology) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+  metadata = book.metadata
+
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book)
+  BakeUnitTitle.v1(book: book)
+  BakeChapterIntroductions.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+  BakeAutotitledNotes.v1(book: book, classes: %w[visual-connection interactive evolution career
+                                                 scientific everyday])
+  book.chapters.each do |chapter|
+    BakePageAbstracts.v1(chapter: chapter)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata, klass: 'summary')
+    chapter.search('section.summary').each do |summary| # TODO: remove this
```

Comment:
IDK if you want to do this now or later

Reply:
this is a bug that SOMEONE (me) introduced in v9. natalia's handling it, but it might be a little while, so I figured we could go ahead with the fix